src/replicache.ts

```diff
@@ -404,11 +407,14 @@ export default class Replicache implements ReadTransaction {
     }
 
     if (reauth && this.getDataLayerAuth) {
+      if (maxAuthTries === 0) {
```

Comment:
Can we make this fail less angrily? Throwing an exception makes it look broken, where the reality is simply that we are logged out. Using @phritz' log level rubric, this seems like an 'info' level log line.

Reply:
We can change to info. np.